feature/gribi/otg_tests/gribi_static_route_on_lemmings_test/gribi_static_route_on_lemmings_test.go

```diff
@@ -0,0 +1,188 @@
+package basic_static_route_support_test
```

Comment:
I suspect this is a already running gribi test on lemmings ....

Lemmings SKU defined here -- http://shortn/_qSGuvLKhMn

Also the gribi calls are same as file which does similar test -- http://shortn/_ghVJjmX5gu

might be we just need to validate and confirm that the existing test works. 

@dplore any thoughts here?